subscription-manager.spec

```diff
@@ -216,6 +229,10 @@
 %global exclude_packages %{exclude_packages}*.plugin.container,}
 %endif
 
+%if !%{use_subscription_manager_migration}
+%global exclude_packages %{exclude_packages}subscription_manager.migrate,
```

Comment:
The `subscription_manager.migrate` is added to excluded packages here.

Reply:
Perfect!